WIP Fix display in development section section

Naylin15 opened the pull request:
### Changes description
Add readme/documentation to ci folder
More changes to come
<!-- Describe results, user mentions, screenshots, screencast (gif) -->

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### References

<!-- issues related (for reference or to be closed) and/or links of discuss -->

Closes #295

Naylin15 commented:
I'm configuring circle ci to make the tests in my fork

Naylin15 commented:
Making the changes & tests to rename the folders to normalize the development section on the project sites

Naylin15 commented:
Depends on #466

ajsb85 commented:
Depends on #466

Naylin15 commented:
We must remove the header for the ChangeLog, it is added to it as a paragraph

With header:

![captura de pantalla 2018-06-01 a la s 1 17 14 p m](https://user-images.githubusercontent.com/30867977/40855671-2a9ddcb8-65a3-11e8-9fc7-a87f89a6e28d.png)

Without header:

![captura de pantalla 2018-06-01 a la s 1 17 22 p m](https://user-images.githubusercontent.com/30867977/40855672-2d0e01c6-65a3-11e8-9da9-24f762eae909.png)

Naylin15 commented:
I'm having issues with the version of the gems to see the display on the project site locally

![captura de pantalla 2018-06-04 a la s 9 38 08 a m](https://user-images.githubusercontent.com/30867977/40920790-94d6afb2-67db-11e8-806c-ecfaae1e3fbf.png)
![captura de pantalla 2018-06-04 a la s 9 38 00 a m](https://user-images.githubusercontent.com/30867977/40920791-950446c0-67db-11e8-86ec-3264f66b990e.png)

Naylin15 commented:
Apparently this problem occurs since commit 6300f0c when running the server at a0bcbbc it builds the site locally

![captura de pantalla 2018-06-04 a la s 5 54 20 p m](https://user-images.githubusercontent.com/30867977/40943946-f7b20b78-6820-11e8-88ba-6e2e9d657b58.png)

Naylin15 commented:
The changelog is not being generated properly, it's currently comparing the last released version with the new versions each time there is a new release, to make it more explicit

Right now it compares version 0.99.46 - 2.0.0, next 0.99.46 - 2.1.0, and so on until 3.0.0

It should be 0.99.46 - 2.0.0, next 2.0.0 -2.1.0

Naylin15 commented:
I've been making tests locally and with CI on my fork to tests the Changelog generation

Naylin15 commented:
After some testing the changelog will be generated as it should
<a name="2.30.0"></a>
# [2.30.0](https://github.com/flyve-mdm/flyve-mdm-android-agent/compare/2.29.1...2.30.0) (2018-06-07)


### Features

* **test:** to drop ([9accf82](https://github.com/flyve-mdm/flyve-mdm-android-agent/commit/9accf82))



<a name="2.29.1"></a>
## [2.29.1](https://github.com/flyve-mdm/flyve-mdm-android-agent/compare/0.99.46...2.29.1) (2018-06-07)

Now there are some issues with a script

Naylin15 commented:
The push of the tags is now working as it should 👍

Naylin15 commented:
I've been working with the release and fixing small issues that arised regarding the changelog

ajsb85 commented on package.json:
```diff
@@ -21,13 +21,13 @@
+    "url": "https://github.com/flyve-mdm/android-mdm-agent/issues"
   },
-  "homepage": "https://github.com/flyve-mdm/flyve-mdm-android-agent#readme",
+  "homepage": "https://github.com/flyve-mdm/android-mdm-agent#readme",
```
http://flyve.org/android-mdm-agent/

Naylin15 commented:
Now the release is working as it should

I have to change the var $GH_TOKEN to $GITHUB_TOKEN since the library node-github-release requires it specifically like that

I'll change the rest of the GH_ variables to keep consistency in the project

Naylin15 commented:
Hi, @ajsb85 @rafaelje 

This PR is ready too

I added the variables in Circle CI with the new names for consistency, they have the same values, I'll remove the old ones after this merge is done

To delete:

GH_USER
GH_TOKE
GH_EMAIL

Renamed to:

GITHUB_USER
GITHUB_TOKEN
GITHUB_EMAIL

Thank you
